lib/webauthn/fake_authenticator.rb

```diff
@@ -25,7 +25,10 @@ def make_credential(rp_id:, client_data_hash:, user_present: true, user_verified
       ).serialize
 
       credentials[rp_id] ||= {}
-      credentials[rp_id][credential_id] = credential_key
+      credentials[rp_id][credential_id] = {
+        credential_key: credential_key,
+        sign_count: 1
```

Comment:
I found it a bit confusing that `sign_count` is set to `1` here, but it ends up returning `0` inside the Attestation Object being returned in the next line.

Reply:
If `sign_count` is being stored in this credentials hash, maybe cleaner to have this value as the only "source of truth", and always pass its value to the `FakeAuthenticator::AuthenticatorData`, instead of relying on a default argument value in some cases?